misk/src/main/kotlin/misk/web/jetty/WebActionsServlet.kt

```diff
@@ -59,6 +59,13 @@ internal class WebActionsServlet @Inject constructor(
     }
   }
 
+  override fun service(request: HttpServletRequest?, response: HttpServletResponse?) {
+    if (request?.method == "PATCH" && response != null) {
+      doPatch(request, response)
+    }
+    super.service(request, response)
```

Comment:
should we rename `handleCall` to `service` and remove the doGet/doPost/doDelete boilerplate?

Reply:
Yeah maybe, though follow-up PR?